compiler/luci/pass/src/FuseAddWithFullyConnectedPass.cpp

```diff
@@ -40,71 +93,21 @@ namespace
  *                |
  *
  */
-bool fuse_add_with_fc(luci::CircleFullyConnected *fc)
+bool fuse_add_with_fc(luci::CircleAdd *add)
 {
-  if (not fc)
-    return false;
-
-  if (fc->dtype() != loco::DataType::FLOAT32)
-    return false;
-
-  if (fc->fusedActivationFunction() != luci::FusedActFunc::NONE)
-    return false;
-
-  auto weights = dynamic_cast<luci::CircleConst *>(fc->weights());
-  if (not weights)
-    return false;
-
-  // Get add node
-  auto fc_output = loco::succs(fc);
-  if (fc_output.size() != 1)
-    return false;
-
-  auto add = dynamic_cast<luci::CircleAdd *>(*fc_output.begin());
-  if (not add)
-    return false;
-  if (add->dtype() != loco::DataType::FLOAT32)
-    return false;
-
-  // Get addition
-  auto addition = add->x() == fc ? dynamic_cast<luci::CircleConst *>(add->y())
-                                 : dynamic_cast<luci::CircleConst *>(add->x());
-
-  // Non-const addition
-  if (not addition)
-    return false;
-
-  auto rank = addition->rank();
-  // TODO Support scalar addition
-  if (rank == 0)
-    return false;
+  PatternNodes nodes;
```

Comment:
it seems you want to find members of this `nodes`.
there are similar styles in other Pass that you can refer.
plz check `compiler/luci/pass/src/FuseRmsNormPass.cpp` that was recently added.